### PR TITLE
Add doc for global Jib configuration

### PR DESCRIPTION
**Pull request by chanseokoh**

### CODE REVIEW

**Comment by chanseokoh** [comment]
This one's ready for review, but #2996 should go in first.

**Comment by chanseokoh** [comment]
Oops, not this one.

File: docs/faq.md [hunk 1]
```diff
@@ -28,7 +28,9 @@ If a question you have is not answered below, please [submit an issue](/../../is
 [How can I inspect the image Jib built?](#how-can-i-inspect-the-image-jib-built)\
 [I would like to run my application with a javaagent.](#i-would-like-to-run-my-application-with-a-javaagent)\
 [How can I tag my image with a timestamp?](#how-can-i-tag-my-image-with-a-timestamp)\
-[How do I specify a platform in the manifest list (or OCI index) of a base image?](#how-do-i-specify-a-platform-in-the-manifest-list-or-oci-index-of-a-base-image)
+[How do I specify a platform in the manifest list (or OCI index) of a base image?](#how-do-i-specify-a-platform-in-the-manifest-list-or-oci-index-of-a-base-image)\
+[I am hitting the Docker Hub rate limiting. How can I configure registry mirrors?](#i-am-hitting-the-docker-hub-rate-limiting-how-can-i-configure-registry-mirrors)\
```
Comment (loosebazooka) [inline docs/faq.md:32]:
```suggestion
[I am hitting Docker Hub rate limits. How can I configure registry mirrors?](#i-am-hitting-docker-hub-rate-limits-how-can-i-configure-registry-mirrors)\
```

File: docs/faq.md [hunk 2]
```diff
@@ -537,6 +539,15 @@ $ docker manifest inspect openjdk:8
 }
 ```
 
+### I am hitting the Docker Hub rate limiting. How can I configure registry mirrors?
```
Comment (loosebazooka) [inline docs/faq.md:542]:
```suggestion
### I am hitting Docker Hub rate limits. How can I configure registry mirrors?
```

**Review by loosebazooka — APPROVED** [review]

**Pull request merged**